int16 LS_LR
int16 LS_UD
int16 LT
int16 RS_LR
int16 RS_UD
int16 RT
int16 DPAD_LR
int16 DPAD_UD

uint8 A
uint8 B
uint8 X
uint8 Y
uint8 LB
uint8 RB
uint8 SELECT
uint8 START
uint8 XBOX
uint8 LS
uint8 RS
